scripts/bench-performance.sh

```diff
@@ -0,0 +1,24 @@
+#! /bin/bash
+#set -eo
+
+echo $PULL_BASE_SHA
+if [[ ! -v PULL_BASE_SHA ]]; then
+    echo "PULL_BASE_SHA is not set"
+    exit 1
+fi 
+if [[ ! -v PULL_PULL_SHA ]]; then
```

Comment:
I'd use `PULL_HEAD_SHA` instead of `PULL_PULL_SHA`. The former is more common terminology

Reply:
I deleted this file because I think it makes more sense to work with this in a separate PR (#561). I also think the terminology is weird but it's the value I got when I followed [these instructions](https://github.com/istio/istio/wiki/Working-with-Prow#writing-tests) to run my tests locally.

Reply:
Fair; consider this one non blocking. It's up to you!

Reply:
I feel like the way to go is to put the SHA's as arguments, but I'm not sure if that will work well with prow. Will investigate